Library/Formula/librsync.rb

```diff
@@ -15,24 +15,14 @@ class Librsync < Formula
 
   option :universal
 
+  depends_on "cmake" => :build
   depends_on "popt"
 
   def install
     ENV.universal_binary if build.universal?
+    args = std_cmake_args
 
-    ENV.append "CFLAGS", "-std=gnu89"
-
-    system "./configure", "--disable-debug",
-                          "--disable-dependency-tracking",
-                          "--prefix=#{prefix}",
-                          "--mandir=#{man}",
-                          "--enable-shared"
-
-    inreplace "libtool" do |s|
-      s.gsub! /compiler_flags=$/, "compiler_flags=' #{ENV.cflags}'"
-      s.gsub! /linker_flags=$/, "linker_flags=' #{ENV.ldflags}'"
-    end
-
+    system "cmake", ".", *args
```

Comment:
Unless there are additional args this should just be `system "cmake", ".", *std_cmake_args` really.